packages/million/dom.ts

```diff
@@ -227,4 +227,13 @@ export const setAttribute = (
   } else {
     removeAttribute$.call(el, name);
   }
+
+  const isInput = el instanceof HTMLInputElement;
+  const isSelect = el instanceof HTMLSelectElement;
+  const isTextArea = el instanceof HTMLTextAreaElement;
+
+  if (name === 'value' && (isInput || isSelect || isTextArea)) {
```

Comment:
@ologbonowiwi I don't think we can directly use this to determine the update of the value attribute. For example, the `<option>` tag also has a value attribute. Here, I suggest using `el.hasOwnProperty('value')` or, even more boldly
```js
if (el.hasOwnProperty(name)) {
    el[name] = value;
}
```
What do you think? ( I am not the maintainer.)

Reply:
you're right, but usually, we update `option` element values through `select` instead of via `option` itself. 

we indeed can have more cases of this issue, but we can wait to have more issues and then add to the conditions rather than over-optimize something that solves the original issue :eyes:

let me know your thoughts

Reply:
I think you're right. I'm wondering if there's a method that can consider all the cases. Currently, I don't have any good ideas either. Besides the `value` property, I also realized that the `textContent` property might have the same issue.